docs/radicle.md

```diff
@@ -28,39 +25,50 @@ while offering global decentralized repository storage through the peer-to-peer
 networking layer.
 
 Since all data on the network is stored locally by peers on the network,
-develops can share and collaborate on Git repositories *without* relying on
+developers can share and collaborate on Git repositories *without* relying on
 intermediaries such as hosted servers.
 
-For more on Radicle Link, check out the [Understanding Radicle][wh] section.
+The radicle-link protocol distinguishes between two types of identities: personal and project.
+The first describes an actor in the system, while the second describes a (software) project
+on which one or more actors collaborate. In Radicle:
 
-## How is Radicle different from GitHub?
+1. peers follow other peers.
+2. peers track projects they are interested in.
+3. peers gossip about projects. This means tracking and replicating objects from
+the peers they follow, about the projects they are interested in.
```

Comment:
I'm realizing this is slightly confusing.... Isn't it more like:

1. Peers track projects they are interested in 
2. Peers track other peers by adding them as remotes to projects
3. Peers gossip about projects. This means they replicate objects from the peers they follow and make it available to the peers that follow them.

@xla

Reply:
Today, what you have @abbey-titcomb is more accurate. happy to adjust.

Reply:
Looks good @abbey-titcomb